contributing.md

```diff
@@ -0,0 +1,81 @@
+# Contributing to Probably
+
+Firstly, thank you for taking an interesting in contributing! Probably is an open-source project, and welcomes
+contributions in the form of feature code, bug reports & fixes, tests, feature suggestions and anything else
+which may help to make it better software.
+
+## Developing with Fury
+
+Probably is developed using [Fury](https://github.com/propensive/fury), which may be installed by running the
+launcher script in the root directory, like so:
+```
+./fury system install
+```
+
+If you do not wish to _install_ Fury, the same script may be used to _run_ Fury without modifying any of your
+system settings.
+
+### Compiling
+
+The simplest way to compile Probably is to run,
+```
+fury
+```
+however, useful variations include,
+- `fury -w`: watch for source-file changes, and recompile as necessary
+- `fury build console -I`: compile and launch the Scala REPL, ignoring any compile errors
+- `fury -F -f <dir>`: compile and save a fat JAR file
+- `fury test`: compile and run the test suite
+
+Fury is not yet stable, so please [report any issues](https://github.com/propensive/fury/issues/) relating to
+Fury.
+
+We recommend using [Visual Studio Code](https://code.visualstudio.com/) with the
```

Comment:
The latest version of IntelliJ with the Scala plugin also works surprisingly well with Fury.

Reply:
I will probably add a mention in this text "at source" the `propensive/admin` repository, which is used to generate this document for each project I have.